kombu/async/aws/sqs/message.py

```diff
@@ -19,7 +19,7 @@ class AsyncMessage(BaseAsyncMessage):
 
     def encode(self, value):
         """Encode/decode the value using Base64 encoding."""
-        return base64.b64encode(value).decode('utf-8')
+        return base64.b64encode(value.encode()).decode()
```

Comment:
You are right. I think there is a suitable function that also handles Python 2/3 compatibility:

```
from kombu.utils.encoding import str_to_bytes
```